### PR TITLE
Metrics doc

### PR DESCRIPTION
**Pull request by iGxnon**
Please briefly answer these questions:

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)

- Change quick start
- Fix benchmark

* what changes does this pull request make?

* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)

### CODE REVIEW

**Review by Phoenix500526 — DISMISSED** [review]
LGTM

**Comment by iGxnon** [comment]
/cc @bsbds 
Could you take a look?

File: doc/metrics.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+3. `learner_promote_succeed`: Counter
+The total number of successful learner promotions while this member is leader.
+
+4. `heartbeat_send_failures`: Counter
+The total number of leader heartbeat send failures (likely overloaded from slow disk).
+
+5. `apply_snapshot_in_progress`: UpDownCounter
+not equals to 0 if the server is applying the incoming snapshot. 0 if none.
+
+1. `proposals_committed`: ObservableGauge
```
Comment (rogercloud) [inline doc/metrics.md:22]: why are the bullet numbers below all 1s?

File: doc/quick-start/Dockerfile [hunk 2]
```diff
@@ -1,25 +0,0 @@
-FROM rust:1.67.1-slim-bullseye as builder
```
Comment (rogercloud) [inline doc/quick-start/Dockerfile:1]: Is this file depracated?

File: scripts/prometheus.yml [hunk 3]
```diff
@@ -6,10 +6,9 @@ scrape_configs:
     static_configs:
       - targets:
           [
-            "172.20.0.2:2379",
-            "172.20.0.3:2379",
-            "172.20.0.4:2379",
-            "172.20.0.5:2379",
+            "node1:9100",
```
Comment (rogercloud) [inline scripts/prometheus.yml:9]: Where do we define these names? on the test servers?

File: doc/quick-start/Dockerfile (hunk 2 above)
Comment (iGxnon) [reply doc/quick-start/Dockerfile:1]: yes

File: doc/metrics.md (hunk 1 above)
Comment (iGxnon) [reply doc/metrics.md]: Sorry, I didn't notice that Markdown rendered it correctly. It is fixed.

File: scripts/prometheus.yml (hunk 3 above)
Comment (iGxnon) [reply scripts/prometheus.yml:9]: Containers on the same network can access each other by using the other container's name.

**Review by rogercloud — APPROVED** [review]

**Comment by Phoenix500526** [comment]
@Mergifyio rebase

**Comment by iGxnon** [comment]
rebased manually

**Review by Phoenix500526 — APPROVED** [review]
LGTM

**Review by rogercloud — APPROVED** [review]

**Review by Phoenix500526 — APPROVED** [review]

**Review by rogercloud — APPROVED** [review]

**Pull request merged**